benchmark/torch/RL4LMs/train.py

```diff
@@ -0,0 +1,173 @@
+        prompt_truncation_side=model_config["args"]["prompt_truncation_side"],
+        generation_kwargs=model_config["args"]["generation_kwargs"])
+    alg_config = config["agent"]["alg"]
+    rl4lm_alg = RL4LMsPPO(
+        model=rl4lms_model,
+        initial_lr=alg_config["args"]["initial_lr"],
+        entropy_coef=alg_config["args"]["entropy_coef"])
+    agent_config = config["agent"]
+    agent = RL4LMsAgent(
+        rl4lm_alg,
+        n_epochs=agent_config["args"]["n_epochs"],
+        batch_size=agent_config["args"]["batch_size"],
+    )
+
+    buffer_config = config["rollout_buffer"]
+    rollout_buffer = DictRolloutBuffer(
+        buffer_size=buffer_config["args"]["n_steps_per_episode"] * instructor_group.n_instructors,
+        observation_space=instructor_group.observation_space,
+        action_space=instructor_group.action_space,
+        device=device,
+    )
+    rollout_util = RolloutUtil(config["kl_div"])
+
+    n_iters = int(config["train_evaluation"]["n_iters"])
+    n_steps_per_iter = instructor_group.n_instructors * buffer_config["args"]["n_steps_per_episode"]
+
+    # gen kwargs for evaluation
+    examiner_config = config["examiner"]
+    # metrics
+    metrics = build_metrics(examiner_config["metrics"])
+    examiner = Examiner(
+        tokenizer=tokenizer,
+        eval_batch_size=examiner_config["args"]["eval_batch_size"],
+        max_prompt_length=examiner_config["args"]["max_prompt_length"],
+        eval_gen_kwargs=examiner_config["args"]["generation_kwargs"],
+        metrics=metrics,
+        samples_by_split=samples_by_split,
+    )
+
+    iter_start = 0
+    examiner.evaluate(policy=agent.alg.model, sample_name_list=["val", "test"], epoch=iter_start)
+
+    epoch = 0
+    for epoch in range(iter_start, n_iters):
+        print("========== BEGIN ==========")
+        print(f"outer epoch: {epoch} / {n_iters - 1}")
+        print("========== BEGIN ==========")
+        outer_start_time = time.time()
+
+        num_timesteps = 0
+
+        while num_timesteps < n_steps_per_iter:
+            run_timesteps = rollout_util.collect_rollouts(agent, instructor_group, rollout_buffer)
+            num_timesteps += run_timesteps
+            agent.learn(rollout_buffer)
+
+        outer_end_time = time.time()
+        print("========== END ==========")
+        print(f"outer epoch: {epoch} / {n_iters - 1}")
+        print(f"time used: {outer_end_time - outer_start_time} second(s), left time:"
+              f"  {1.0 * (outer_end_time - outer_start_time) * (n_iters - epoch - 1) / 60 / 60} hour(s)")
+        print("========== END ==========")
+
+        # evaluate on val set in the given intervals
+        if (epoch + 1) % config["train_evaluation"]["eval_every"] == 0:
+            examiner.evaluate(policy=agent.alg.model, sample_name_list=["val"], epoch=epoch)
+
+    examiner.evaluate(policy=agent.alg.model, sample_name_list=["val", "test"], epoch=epoch)
```

Comment:
remove this evaluation. As we can evaluate the performance of the converged model via line 136~137, by setting `(epoch + 1) % config["train_evaluation"]["eval_every"] =0 `